README.md

```diff
@@ -1 +1,46 @@
-# hw01
\ No newline at end of file
+# hw01
+title: "Hw 1"
```

Comment:
If you want these on separate lines, you need a line break between them

```
title: "Hw 1"

author: Zoe Cullison-Shimada  
```

title: "Hw 1"

author: Zoe Cullison-Shimada